module2/projects/coupon-codes-api/evaluation.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Coupon Codes API Evaluation
+---
+
+## Evaluation Criteria
+
+_[Back to Coupon Codes Home](./index)_
+
+## Presentation
+For the project evaluation, you will meet with an instructor for a 20 minute session. You should prepare a presentation that covers the following:
+
+<!-- TO-DO Change to using FE to demonstrate functionality? -->
+* Demonstration of functionality via Postman suites
+* Discussion of technical quality and organization of the code, identifying code that should be refactored and how it would be refactored
+* Running your application's test suite and a discussion of test coverage (happy/sad paths and any edge cases)
+* Identifying the area(s) of code of which you are most proud, and an area where you would like specific feedback
+
+Students should focus on practicing technical communication that is succinct and utilizes appropriate technical vocabulary.
+
+Slides are not required, but encouraged as a way to facilitate the presentation along with sharing specific code examples.
+
+With remaining time, instructors will ask questions and follow-up on feedback requests.
+
+
+## Rubric
+Your project will be evaluated based on the following rubric: 
+
+### Feature Delivery
+
+* Exceeds Expectations: Project completes all requirements and at least one extensions.
+* Meets Expectations: Project completes all requirements. Postman tests pass, and FE functions correctly when wired up to API. 
+* Approaching Expectations: Project fails to complete 1 - 2 required endpoints
+* Below Expectations: Project fails to complete more than 3 or more endpoints
+
+### Project Management
+
+* Exceeds Expectations: GitHub Project Board is fully up to date in all checkins and the evaluation. Live presentation is prepared with visuals, a logical flow, and distributed talking points. Students complete all requirements in "Meets Expectations" as well
+* Meets Expectations: Students use GitHub projects or similar tool to track all project tasks. Project board is mostly up to date in all checkins, and fully up to date in evaluation. Students participate in meaningful code review and it's visible through PR comments or live meetings (students can speak to one of these meetings). Live presentation is prepared, and follows a logical flow. All group members participate, but maybe with some instructor prompting.
```

Comment:
Do you think we should evaluate students on project management for a solo project? Do we currently do that? I think evaluation of some planning would be good but not sure what makes the most sense for solo.

Reply:
Looks like we currently evaluation on technical presentation instead of project management here -> https://backend.turing.edu/module2/projects/coupon_codes/evaluation

I think I'm I'd learn more towards technical presentation rather than proj management for this project

Reply:
Good idea that makes a lot of sense. I largely copied over the rubric from the group project to get this up for feedback. I will update to evaluate the presentation instead. FWIW I think it's okay and good for students to practice PM on solo projects but given the timeline for this one, it may end up just feeling like overhead for them.

Reply:
Totally -- agreed!